src/api/faucet.ts

```diff
@@ -24,7 +24,11 @@ export class Faucet {
    * @param timeoutSecs Timeout in seconds. Defaults to 20 seconds.
    * @returns Hashes of submitted transactions
    */
-  async fundAccount(args: { accountAddress: HexInput; amount: number; timeoutSecs?: number }): Promise<Array<string>> {
+  async fundAccount(args: {
+    accountAddress: AccountAddress;
```

Comment:
is there a reason we need the AccountAddress and not HexInput?

Reply:
I will change this back for now.

But my thought was, if we accept `HexInput` on these type of method, then we will have to do address validation check on every methods.